<launch>

  <arg name="gpu" default="-1" />
  <arg name="input_image" />
  <arg name="ssd_params_path" />
  <arg name="tweet_topic_name" default="/tweet" />

  <node name="ssd_object_detector"
        pkg="jsk_perception" type="ssd_object_detector.py"
        output="screen" >
    <remap from="~input" to="$(arg input_image)" />
    <rosparam file="$(arg ssd_params_path)" command="load" />
    <rosparam subst_value="true">
      gpu: $(arg gpu)
    </rosparam>
  </node>

  <node name="mail_notify"
        pkg="jsk_patrol" type="patrol_mail_notify_node.py" >
    <remap from="~input/original_image" to="$(arg input_image)" />
    <remap from="~input/image" to="ssd_object_detector/output/image" />
    <remap from="~input/class" to="ssd_object_detector/output/class" />
    <remap from="~tweet" to="$(arg tweet_topic_name)" />
  </node>

</launch>
